docs/src/operate/security/cloud-authentication.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Cloud Authentication
 tags: [Operate, Security]
-sidebar_position: 3
```

Comment:
Question: why removing the `sidebar_position` settings?

In some cases, I agree, there is no specific reader order. But for the *Security* section the alphabetic order makes little sense to me - jumping back and forth over device / cloud / service related topics.

Reply:
I found the order very difficult to find things as you'd have to scan the list going from top to bottom looking for that page you're interested in.

Though looking at the list again, I think we can do better group of the pages (but probably best done in a follow up PR). But for example we have two device certificate pages, and then we could also group the HTTP and MQTT pages together under "Local Endpoints" (or something like that)